### DISCUSSION TITLE
What's the point of this package?

### DISCUSSION THREAD

**Discussion by monbrey** [Mateo-tem/discord-modals#15] [Q&A] [answered]
Why does this exist as a third party addon to discord.js (comprised mostly of code copied from that library) when discord.js will have first-party support for Modals in both v13.7 and v14.0 releases?

**Comment by Mateo-tem** [comment, marked as answer]
**Well, first of all, thank you for asking.** 

I want to say that this package has no intention of copying discord.js and I apologize if you think that happens.
Mainly, my motivation to make this package was because there are people who want to try the new modals right now, I see people asking for when the modals will be in discord.js and I feel that those people need something like this, it's true that it may be something useless for some people because discord.js will release it as soon as all the work for those versions (v13.7 and v14) will be finished, I know there are other methods (and even better), but I just release this package in case there are people who need it, I have no other intentions apart from that. It's the first package I make, and I just wanted to share it. I got some code base from discord.js to adapt it, but I'm trying to make this package as original as possible. This is not the only package. Actually, I think there are four such packages.
